.github/ISSUE_TEMPLATE/new-feature-or-discussion.md

```diff
@@ -0,0 +1,17 @@
+---
+name: Request a new GEOS-Chem feature or start a discussion
+---
+
+<!--- Before submitting an issue please see:
+
+- [Contributing guidelines](https://geos-chem.readthedocs.io/en/stable/reference/CONTRIBUTING.html)
+
+--->
+
+### Your name and institution
+
+We ask that users identify themselves and their institutions when submitting a new issue.
```

Comment:
This could be even more concise by making the title "Name and Institution (Required)" followed by,
Name:
Institution:

Reply:
Now done in commit 581e361a